web/unit_tests/base.conf.js

```diff
@@ -37,29 +37,27 @@ module.exports = {
-    'web/unit_tests/dev_resources.js',          // Defines com.keyman.dom objects separate from KMW for unit testing.
-    'common/web/recorder/build/index.js',
-    'web/unit_tests/recorder_InputEvents.js',   // The object definitions used to generate/replicate key events for engine tests.
-                                                // Includes KMW's Device class, which is used by test_utils below.
-    'web/unit_tests/test_init_check.js',        // Ensures that tests will initialize properly
-    'common/test/resources/timeout-adapter.js', // Handles configuration timeout setup at runtime.
-    'web/unit_tests/test_utils.js',             // A basic utility script useful for constructing tests
-    'web/unit_tests/cases/**/*.js',             // Where the tests actually reside.
-    'common/test/resources/json/**/*.json',     // Where pre-loaded JSON resides.
+    'web/unit_tests/modernizr.js',               // A dependency-managed utility script that helps with browser feature detection.
+    'web/tools/element-wrappers/build/index.js', // Defines com.keyman.dom objects separate from KMW for unit testing.
+    'web/tools/device-detect/build/index.js',    // Defines com.keyman.utils.Device, separated from KMW for use in unit test setup.
+    'web/tools/recorder/build/index.js',         // The object definitions used to generate/replicate key events for engine tests.
+                                                 // Includes KMW's Device class, which is used by test_utils below.
+    'web/unit_tests/test_init_check.js',         // Ensures that tests will initialize properly
+    'common/test/resources/timeout-adapter.js',  // Handles configuration timeout setup at runtime.
+    'web/unit_tests/test_utils.js',              // A basic utility script useful for constructing tests
+    'web/unit_tests/cases/**/*.js',              // Where the tests actually reside.
+    'common/test/resources/json/**/*.json',      // Where pre-loaded JSON resides.
     {pattern: 'common/test/resources/fixtures/**/*.html', watched: true}, // HTML structures useful for testing.
     {pattern: 'common/test/resources/**/*.*', watched: true, served: true, included: false}, // General testing resources.
     {pattern: 'web/release/unminified/web/**/*.css', watched: false, served: true, included: false}, // OSK resources
     {pattern: 'web/release/unminified/web/**/*.gif', watched: false, served: true, included: false}, // OSK resources
     {pattern: 'web/release/unminified/web/**/*.png', watched: false, served: true, included: false}, // OSK resources
-    {pattern: 'web/release/unminified/web/**/*.eot', watched: false, served: true, included: false}, // OSK resources
     {pattern: 'web/release/unminified/web/**/*.ttf', watched: false, served: true, included: false}, // OSK resources
-    {pattern: 'web/release/unminified/web/**/*.woff', watched: false, served: true, included: false}, // OSK resources
```

Comment:
We may include .woff files in the future, although .eot are thankfully dead. So perhaps we should leave this line in?

Reply:
When executing unit tests, we get a runtime warning about there being no match.  I figured I'd eliminate the warning.